CiarnuroRT/webapp/style.css

```diff
@@ -414,5 +414,13 @@ div.banner{
 
 #playerEven:nth-child(even) {
     background-color: rgba(128, 128, 128, 0.4);
-    z-index: -1; 
+    z-index: -100; 
```

Comment:
Is this a typo or are you having problems with the alternating lines?

Reply:
Oh sorry, I didn't notice that I changed the wrong code. I'll fix my mistake right away